### ISSUE TITLE
[Feature Request] Add custom key and custom value on podserver

### ISSUE THREAD

**Issue by johnongit** [btcpayserver/btcpayserver-plugins#60]
Podcast2 namespace can use TVL custom field. This field can be used on the receiver side to handle podcast contributor.
This is useful for hoster to provide custody for contributor (like my podcast).

https://github.com/satoshisstream/satoshis.stream/blob/main/TLV_registry.md

**Comment by dennisreimann** [comment]
Yes, this will come, especially as I intend to connect PodServer and LNbank.
